src/self_update.sh

```diff
@@ -155,6 +159,11 @@ function parse_self_update_options()
 
 function self_update_help()
 {
+  if [[ "$1" == --help || "$1" == -h ]]; then
```

Comment:
You don't want `-h` in this condition, otherwise, we will open the man page for the `-h` option. The `-h` option should be the short summary of the feature.